### DISCUSSION TITLE
How to set **minimum** resampling resolution?

### DISCUSSION THREAD

**Discussion by sarthakpati** [TorchIO-project/torchio#835] [Q&A] [answered]
Hi,

I know how to set the target resolution of the resampler, but it is possible to set the **minimum** resolution for all available axes, instead? For example, if I apply the minimum resolution of `1` to an image of resolution of `[0.5,0.5,5]`, it should get resampled to `[0.5,0.5,1]`...

Cheers,
Sarthak

**Comment by fepegar** [comment, marked as answer]
I think you mean "spacing". The resolution is high when the spacing is low.

That feature is currently not implemented, but it should be straightforward to write a transform that inherits from `Resample` and chooses the target spacing according to the image spacing.

**Reply by sarthakpati** [reply]
Yes, I do mean spacing. 

Anyway, I was hoping it was present before I started writing something by myself. No worries, thanks.
